src/stylus/components/_date-picker-title.styl

```diff
@@ -26,6 +26,7 @@ rtl(v-date-picker-title-rtl, "v-date-picker-title")
     font-weight: 500
     position: relative
     overflow: hidden
+    padding-bottom: 2px
```

Comment:
Some alphabets have letters that need more than 2px, please change it to 8px (should be enough) and also add negative margin so that the title height remains unchanged

Reply:
Could you elaborate on the negative margin? :)